io/net/scp_ssh.py

```diff
@@ -84,7 +84,7 @@ def test_scpandssh(self):
         '''
         check scp and ssh
         '''
-        cmd = "ssh %s@%s \"echo hi\"" % (self.user, self.peer)
+        cmd = "ssh -o StrictHostKeyChecking=no %s@%s \"echo hi\"" % (self.user, self.peer)
```

Comment:
@abdhaleegit Readme says ssh key to peer needs to be generated. Is that not enough ?
https://github.com/abdhaleegit/avocado-misc-tests/blob/241858f987421fb923349977686363a858a372c6/io/net/scp_ssh.py.data/README.txt

Reply:
@narasimhan-v  Ah ok it is not enough, thinking of including passwordless ssh setup code in library as it is used in many tests,  instead of manual setting, will that be usefull ?

Reply:
@abdhaleegit not sure we can target this to lib as it a user choice to do that so README i think should be fine

Reply:
@abdhaleegit If that can be handled in each of the tests, then I am fine with that as well.

Reply:
@abdhaleegit But cleaner way is to use a python lib or write an avocado lib :)

Reply:
In my system even after setting passwordless ssh, still I see the test stop here in some case:

Running 'ssh root@140.1.1.122 "rm /tmp/tempfile"'
The authenticity of host '140.1.1.122 (140.1.1.122)' can't be established.
ECDSA key fingerprint is SHA256:sOcszqO0aqSOhaqll0ez29+70StJTRPtTlUX19blPE0.
Are you sure you want to continue connecting (yes/no)?

so when I enter 'yes' the test continued with out password, this was the reason for this patch. 

Having a avocado lib is not a immediate requirement. so can we push this patch for now ?

Reply:
@abdhaleegit Can you please check if passwordless ssh was set using this IP address/hostname ?
Because, we usually send the keys via one IP and try with another IP, which asks for confirmation first time.

If this is reproduced with same IP, then we can look into it.

Reply:
the public keys were copied not with the same IP, but by default IP. 

On the utility part I am planning to add  passwd_less_ssh() to utils/network.py by making use of pexpect library  

This will solve scpssh and bonding test issue, we can close this